main.go

```diff
@@ -4,24 +4,36 @@ import (
 	"flag"
 	"log"
 
-	"read-adviser-bot/clients/telegram"
+	tgClient "read-adviser-bot/clients/telegram"
+	eventconsumer "read-adviser-bot/consumer/event-consumer"
+	"read-adviser-bot/events/telegram"
+	"read-adviser-bot/storage/files"
 )
 
-func main() {
+const (
+	storagePath = "file-storage"
+	batchSize   = 10
+)
 
-	tgClient := telegram.New(mustHost(), mustToken())
+func main() {
 
-	// fetcher = fetcher.New(tgClient)
+	eventsProcessor := telegram.New(
+		tgClient.New("api.telegram.org", mustToken()),
```

Comment:
It’s better to move the telegram host into a function or pass it through an environment variable or flag